<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TCP-KP-RECEIVER by balb0x</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Server Configuration</string>
   </property>
   <widget class="QComboBox" name="ip_combo_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6000</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Server Configuration</string>
   </property>
   <widget class="QTextEdit" name="status_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="relay_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Relay 1</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay_1_image_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="relay_1_image_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="relay_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Relay 3</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay_3_image_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="relay_3_image_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="relay_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Relay 2</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay_2_image_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="relay_2_image_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="relay_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Relay 4</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay_4_image_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="relay_4_image_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input 2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
